Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs

```diff
@@ -366,6 +379,37 @@ private void StartHeartbeatCheckLoop(TimeSpan heartbeatInterval, CancellationTok
 			}, token).ConfigureAwait(false);
 		}
 
+		private void StartTokenRefreshLoop(TimeSpan checkInterval, CancellationToken token)
```

Comment:
It would be nice to have only one loop for all `RelayServerConnection` instances.